portfolio/src/main/webapp/index.html

```diff
@@ -4,23 +4,51 @@
 <head>
 	<meta charset="UTF-8">
 	<title>Azfaar's Portfolio</title>
-	<link rel="stylesheet" href="style.css">
 	<script src="script.js"></script>
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+		integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+	<link
+		href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i|Raleway:300,300i,400,400i,500,500i,600,600i,700,700i|Poppins:300,300i,400,400i,500,500i,600,600i,700,700i"
+		rel="stylesheet">
+	<link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.0-2/css/all.min.css" rel="stylesheet">
+	<link rel="stylesheet" href="style.css">
 </head>
 
-<body>
-	<div id="content">
-		<img src="/images/azfaar.jfif" alt="Azfaar's picture" width="200" height="200">
-		<h2>Hi, my name is</h2>
-		<h1>Azfaar Qureshi</h1>
-		<h3>Welcome to my portfolio.</h3></br>
-		<p>I'm currently a 3rd year Computer Engineering student at the University of Waterloo.
-			I have experience with Frontend, Backend and Production engineering.
-			I'm graduating in April 2022 and am currently searching for my next internship.
-			Take a look at my resume and previous experiences below!</p>
-		<p>Click here to get a random greeting:</p>
-		<button onclick="addRandomGreeting()">Hello</button>
-		<div id="greeting-container"></div>
+<body class="bg">
+	<script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
+		integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous">
+	</script>
+	<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
+		integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous">
+	</script>
```

Comment:
It looks like you aren't using jQuery or Popper anywhere on this page. I'd recommend not including these `script` tags until you need these two libraries on your portfolio.